src/CommentList.re

```diff
@@ -23,19 +23,19 @@ let make = (~story: StoryData.story_with_comments, _children) => {
       };
     | None => collapsed
     };
-  let getCommentIdFromEvent = (event: ReactEventRe.Mouse.t) =>
```

Comment:
I did not made these changes 🤔

Reply:
these changes are created when I edit `CommentList.re` file and hit `ctrl + S`

Reply:
Not sure whether they are expected changes

Reply:
I hope you're using https://github.com/chenglou/upgrade-reason-react and not upgrading manually btw lol. Yes, `ReactEventRe` is deprecated.

Reply:
I use  chenglou/upgrade-reason-react to migrate in the previous PR, not sure why it still cause this issue

Reply:
https://github.com/reasonml-community/reason-react-hacker-news/pull/18

Reply:
Ah weird, maybe I didn't catch this case. Thanks.

Reply:
you're welcome